openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -603,6 +625,141 @@ Location: https://client.example.net/cb?
   &error_description=Unsupported%20response_type%20value
 ```
 
+# Authorization Challenge Endpoint
+
+An Authorization Server MAY publish the `authorization_challenge_endpoint` parameter in its Authorization Server Metadata. In this case, the Wallet SHOULD use this endpoint to obtain authorization. This enables use cases where an Issuer requests Presentation of a Credential before issuing its Credential.
```

Comment:
Need an IANA consideration for registering `authorization_challenge_endpoint`?

Reply:
Or maybe not? We're reusing the endpoint from https://www.ietf.org/archive/id/draft-parecki-oauth-first-party-apps-02.html#section-10.2 - I'm not 100% sure if that's a good idea or not.

Reply:
Referencing a draft could not be the "best" idea, since it could theoretically change?